openfisca_us/variables/finance/tax/outputs.py

```diff
@@ -438,10 +438,37 @@ class c02900(Variable):
     value_type = float
     entity = TaxUnit
     definition_period = YEAR
+    label = "'Above the line' AGI deductions"
+    unit = USD
     documentation = (
         """Total of all 'above the line' income adjustments to get AGI"""
     )
 
+    def formula(tax_unit, period, parameters):
+        misc_haircuts = parameters(period).tax.ald.misc
+        other_deductions = add(tax_unit, period, "c03260", "care_deduction")
+        return (
+            sum(
```

Comment:
Nit but I think it'd be clearer if this part was its own variable, like `other_deductions` so you just have to `return haircut_deductions + other_deductions`

Reply:
OK I've split it up, are the variable names OK?

Reply:
I was thinking Python variables. Right now it's kind of arbitrary which deductions allow haircuts, since some had full haircuts in the past (i.e. they didn't count as deductions against AGI) and some have always been fully deducted.

Rather than adding these variables, how about adding two new parameters for haircuts on `c03260` and `care_deduction`, to make it a single consistent variable?

Reply:
OK, yeah that makes sense - how about now?